packages/x-tailwindcss/src/x-tailwind-plugin/components/sliding-panel/buttons.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * Returns the button position variants for the component `sliding panel`.
+ *
+ * @returns The {@link CssStyleOptions} for the variant.
+ */
+export function slidingPanelButtons() {
+  return {
+    '.sliding-panel-button': {
```

Comment:
Should we split these buttons into different components?
Maybe one for `buttons-center` and another for `buttons-outside`.

Reply:
Hi, I have some doubts here about extracting the hover instead.